File: parent327/pinyougou-web-manager/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){

    //分页
    this.findPage=function(page,rows){
        return $http.get('../seckill/findPage.do?page='+page+'&rows='+rows);
    }
    // //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../seckill/search.do?page='+page+"&rows="+rows,searchEntity);
    }

    this.updateStatus = function(ids,status){
        return $http.get('../seckill/updateStatus.do?ids='+ids+"&status="+status);
    }
});

File: parent327/pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seckill.SeckillGoods;
import entity.PageResult;

public interface SeckillService {

    void updateStatus(Long[] ids, String status);

    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);
}
